css/main.css

```diff
@@ -1,3 +1,878 @@
+}
+
+body {
+    font-family: 'Quicksand', sans-serif;
+}
+
+ul {
+    list-style: none;
+}
+
+li {
+padding-left: 15px;
+}
+
+.side-bars {
+    display: none;
+}
+
+.header-section {
+    position: fixed;
+    overflow: hidden;
+    background-color: white;
+    z-index: 7;
+}
+
+.header-box {
+
+padding-top: 10px;
+
+
+}
+
+.first-section{
+    background-color: rgb(247,247,247);
+}
+.relative{
+    position: relative;
+
+}
+.big-img-ab {
+    z-index: 1;
+    position: absolute center;
+    left: 10px;
+    top: 10px;
+    height: 500px;
+    transform: rotate(-125deg) scaleX(-1) skew(-25deg, 23deg);
+ 
+}
+
+.date-of-year {
+    position: absolute;
+top:150px;
+left:7px;
+z-index: 4;
+transform: rotate(90deg);
+
+}
+
+.hero-image-ab {
+    
+    z-index: 2;
+    position: absolute;
+    left: 10px;
+    top: 114px;
+   
+    
+    
+}
+
+.hero-image-desktop {
+    display: none;
+
+}
+
+.small-img-ab {
+    z-index: 3;
+    position: absolute;
+    left: 4px;
+    top: 82px;
+    transform: rotate(174deg);
+    height: 120px;
+}
+
+.flower {
+    position: absolute;
+    z-index: 4;
+    left:214px;
+    top:308px;
+    transform: rotate(15deg);
+}
+
+.festival-details {
+  
+    position:absolute;
+    z-index: 4;
+    left: 80px;
+    top: 104px;
+
+}
+
+.ticket-button {
+    color: white;
+    background-color: rgb(83, 98, 172);
+    margin-top:15px;
+    font-size: 14px;
+    border: none;
+    padding: 11px 22px;
+    cursor:pointer;
+    z-index: 11;
+    
+
+}
+
+h1, h2, h3, h4, p {
+color: white;
+}
+
+.line-up-heading {
+    font-size: 13px;
+    padding-bottom: 10px;
+}
+
+.line-up-paragraph {
+    font-size: 15px;
+}
+
+.navy-blue-follow {
+    color: #190F3F;
+    padding-bottom:2px;
+}
+
+.navy-blue-h2 {
+     color: #190F3F;
+     padding-bottom:2px;
+
+}
+
+.line-up-section {
+    background-color:#5362AC;
+    padding-top: 32px;
+}
+
+.background-shapes {
+    padding-top: 100px;
+    padding-bottom: 100px;
+    position: relative;  
+}
+
+.navyblue-blob{
+    background-image: url(../img/festival_blob-shape-navyblue\ -\ Copy.svg);
+    background-repeat:no-repeat;
+    background-position: center;
+    background-size: contain;
+    height:240px;
+    transform: rotate(122deg);
+   
+
+}
+
+
+.lily-flower {
+    position: absolute;
+    top:287px;
+    left:-8px;
+    height:102px;
+    transform:rotate(-112deg);
+    
+}
+
+.content {
+    position:absolute;
+    padding-top: 35px;
+    
+}
+
+.section-three {
+
+    background-image: url(https://images.unsplash.com/photo-1514525253161-7a46d19cd819?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1934&q=80);
+    height: 500px; 
+    background-position: center; 
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.flower-shape {
+     background-image: url(../img/festival_lilly-flower.svg);
+     background-repeat: no-repeat;
+     background-size: 63px;
+     height: 66px;
+     width: 50px;
+     left:228px;
```

Comment:
Since you want the flower to be pinned to the right of the brown shape, you can set the `right` property instead of the `left` here. It should give you more control because at the moment the flower is not moving in line with the brown shape when I resize the screen.